### ISSUE TITLE
MSFT-137[Login] Some social media hyperlinks missed by the screen reader

### ISSUE THREAD

**Issue by veshu** [veshu/testjiraimport#76]
**Steps to Reproduce**
1. Go to https://accounts.topcoder.com/member
2. Switch on the screen reader
3. Press Tab Key and Listen

**Expected Result**
It should be read-out by the screen reader.

**Actual Result**
Some social media hyperlinks missed by the screen reader 
 - GITHUB
 - GOOGLE
 - FACEBOOK
 - TWITTER

**Device**: Desktop/Labtop
**Operating System**: Windows
**Browser**: Chrome
**WCAG Category**: 1. Perceivable
**WCAG Level**: A
**WCAG Success Criterion**: 1.3.1 Info and Relationships

**Attachment**
https://topcodermsft-my.sharepoint.com/personal/pd-topcoder_topcodermsft_onmicrosoft_com/Documents/Forms/All.aspx?cid=6aa53cde%2D1eef%2D4bae%2Dbfe5%2Df5f5f79e9df4&FolderCTID=0x0120005C598A51049FC14CBA882E1AEE168F51&sortField=Modified&isAscending=true&id=%2Fpersonal%2Fpd%2Dtopcoder%5Ftopcodermsft%5Fonmicrosoft%5Fcom%2FDocuments%2FMSFT%2DTeams%2DQA%2FJune%202019%2F29%20Jun%2FTest%20Results%2FBug%20Videos%20%26%20Log%20Files%2Fcodejam%2DAccessibility%2F47
